app/web_hub.go

```diff
@@ -161,13 +161,19 @@ func (a *App) GetHubForUserId(userId string) *Hub {
 func (a *App) HubRegister(webConn *WebConn) {
 	hub := a.GetHubForUserId(webConn.UserId)
 	if hub != nil {
+		if metrics := a.Metrics(); metrics != nil {
+			metrics.IncrementWebSocketBroadcastUsersRegistered(strconv.Itoa(hub.connectionIndex), 1)
```

Comment:
Phew .. these method names do get long. What do you think about - "IncrementWebSocketBroadcastRegUsers" ?

Reply:
Yeah, I know. I'm not a big fan but when you read about it in the future the Reg part is not very  readable or if you use that in other method for any reason.

Reply:
Alright.